central/role/service/service_impl.go

```diff
@@ -53,12 +53,13 @@ var (
 			"/v1.RoleService/DeleteSimpleAccessScope",
 		},
 		user.Authenticated(): {
+			"/v1.RoleService/GetMyPermissions",
```

Comment:
similar concern here, however at least here there's no evidence that anyone ever expected this endpoint to be called by service identity

Reply:
No need for `ServiceOr` here: service identities are not associated with any permissions.